README-it.md

```diff
@@ -0,0 +1,464 @@
+# Plugin per notifiche via Email in caso di problemi
```

Comment:
Please add a link in file `README.md` to `README-it.md` as shown below.

`README.md` before change:

```
This page is available in English (this document), [German](README-de.md), [Russian](README-ru.md), and [Slovenian](README-si.md).
```

`README.md` after change:

```
This page is available in English (this document), [German](README-de.md), [Russian](README-ru.md), [Slovenian](README-si.md), and [Italian](README-it.md).
```